tests/trait_mapping/test_create_efo_table.py

```diff
@@ -33,15 +33,25 @@ def test_get_ols_details(self):
             self.assertEqual(len(trait), 22, 'Each output line must contain correct number of columns')
 
         # Trait 1
-        self.assertEqual(trait_1[0], 'Verheij syndrome')  # disease
-        self.assertEqual(trait_1[1], 'inherited genetic disease')  # child of
-        self.assertEqual(trait_1[12], 'OMIM:615583')  # OMIM cross-ref
-        self.assertEqual(trait_1[15], 'UMLS:C3810023')  # ULMS cross-ref
-        self.assertEqual(trait_1[21], 'MONDO_0014263')  # MONDO ref
+        self.assertEqual(trait_1[0], 'gyrate atrophy')                                # disease
+        self.assertIn('unclassified familial retinal dystrophy', trait_1[1])          # child of
+        self.assertEqual(trait_1[7], 'MeSH:C538071 || MeSH:D015799')                  # MeSH cross-refs
+        self.assertEqual(trait_1[8], 'NCIT:C84744')                                   # NCIT cross-ref
+        self.assertEqual(trait_1[12], 'OMIM:258870')                                  # OMIM cross-ref
+        self.assertEqual(trait_1[13], 'DOID:1415')                                    # DOID cross-ref
+        self.assertEqual(trait_1[15], 'UMLS:C0018425')                                # ULMS cross-ref
+        self.assertEqual(trait_1[18], 'Orphanet:414')                                 # Orphanet cross-ref
+        self.assertEqual(trait_1[21], 'MONDO:0009796')                                # MONDO ref
 
         # Trait 2
-        self.assertEqual(trait_2[0], '3-methylglutaconic aciduria type 9')  # disease
-        self.assertEqual(trait_2[1], 'disease')  # child of
-        self.assertEqual(trait_2[3], '3-methylglutaconic aciduria-epilepsy-spasticity-severe intellectual '
-                                     'disability syndrome || MGA9')  # synonyms
-        self.assertEqual(trait_2[18], 'Orphanet:505216')  # Orphanet ref
+        self.assertEqual(trait_2[0], 'Gyrate atrophy of choroid and retina')          # disease
```

Comment:
Oh, so this test now also checks that, even though the terms are considered equivalent (because the xrefs say so), they are not represented in exactly the same way in both ontologies?